lib/AppInfo/Application.php

```diff
@@ -27,17 +27,21 @@
 use OCP\AppFramework\Bootstrap\IBootstrap;
 use OCP\AppFramework\Bootstrap\IRegistrationContext;
 
-class Application extends App implements IBootstrap {
-	public const APP_ID = 'nc_ionos_processes';
+class Application extends App implements IBootstrap
```

Comment:
commented on this in #3 see https://github.com/IONOS-Productivity/nc_ionos_processes/pull/3#discussion_r1810125113

Reply:
Fixed. The `nextcloud/coding-standard` dependency was missing.